dev/source/docs/esp32-autopilot.rst

```diff
@@ -0,0 +1,14 @@
+.. _esp32-autopilot:
+
+=====================
+ESP32 Based Autopilot
+=====================
+
+.. note:: this code and development is experimental. No OEM has produced an ArduPilot compatible autopilot based on the ESP32. This code requires that sensors be attached to an ESP32 board or making a custom PCB. Only the 'classic' ESP32 is currently targeted but work on ESP32S3 in in progress.
+
+.. note:: Monitor the `ArduPilot Discord ESP32 channel <https://discord.com/channels/674039678562861068/777475957476032523>`__ for updates and developments.
+
+Porting ArduPilot to the ESP32
```

Comment:
Please change "Porting ArduPilot to the ESP32" to:  "Compiling ArduPilot for ESP32"